web/cobrands/fixmystreet/assets.js

```diff
@@ -631,7 +656,12 @@ fixmystreet.assets = {
             fixmystreet.map.addControl(fixmystreet.assets.controls[i]);
             fixmystreet.assets.controls[i].activate();
         }
-    }
+    },
+
+    check_layer_relevant: function(layer, category) {
+      return OpenLayers.Util.indexOf(layer.asset_category, category) != -1 &&
+        ( !fixmystreet.bodies.length || !layer.body || OpenLayers.Util.indexOf(fixmystreet.bodies, layer.body ) != -1 );
+    },
```

Comment:
Hmm, bit annoying if our automated JS linter stops you doing an okay loop thing, but misses this, not sure what happened there.